src/Effects/EffectMgr.h

```diff
@@ -73,6 +73,10 @@ class EffectMgr:public Presets
         int     nefx;
         Effect *efx;
         const AbsTime *time;
+        
+        int numerator = 0;
```

Comment:
Should these values be stored in this class or in the child classes?

Reply:
Also if these values are to be stored here, are they being written to XML correctly? I'm not seeing code handling that in EffectMgr.cpp.

Reply:
oh, that was missing. is being added now.